command/genesis/params.go

```diff
@@ -576,6 +583,27 @@ func (p *genesisParams) validateBurnContract() error {
 	return nil
 }
 
+func (p *genesisParams) validateGenesisBaseFeeConfig() error {
+	if p.baseFeeConfig == "" {
+		return errors.New("genesis base fee config must be set")
+	}
```

Comment:
What is the point in insisting `baseFeeConfig` to be mandatory, when it is related to EIP-1559 (which means that if London hard fork is not enabled, we don't need base fee). IMO it should not be a mandatory flag.

On the other hand, I'm not sure should we default it in that case and persist it in the `genesis.json` or completely omit it?

Reply:
`baseFeeConfig` is not mandatory. This is the case when user gives a empty string as input in that case we should log an error and not use default values. Default values will be used in case no `genesis-base-fee-config` flag is used while genesis creation.

The error is string is quite misleading, I will change it.